tasty/source_shapes/haystack/nrel.json

```diff
@@ -2,6 +2,123 @@
   "prefix": "nrel",
   "namespace": "https://project-haystack.org/datashapes/nrel#",
   "shapes": [
+    {
+      "name": "ZoneRelativeHumidityShape",
+      "types": ["air-humidity-sensor"],
+      "tags": ["zone"]
+    },
+    {
+      "name": "BinaryOccupancySensorShape",
+      "types": ["point"],
+      "tags": ["occupied", "sensor"],
+      "tags-custom": ["occupancyindicator"]
+    },
+    {
+      "name": "OccupancyOverrideCommandShape",
+      "types": ["point"],
+      "tags": ["occupied", "cmd"]
+    },
+    {
+      "name": "ZoneTemperatureCoolingEffectiveSetpointShape",
+      "types": ["air-temp-sp"],
+      "tags": ["zone", "cooling", "effective"]
+    },
+    {
+      "name": "ZoneTemperatureHeatingEffectiveSetpointShape",
+      "types": ["air-temp-sp"],
+      "tags": ["zone", "heating", "effective"]
+    },
+    {
+      "name": "ZoneTemperatureCoolingOccupiedSetpointShape",
+      "types": ["air-temp-sp"],
+      "tags": ["zone", "cooling", "occ"]
+    },
+    {
+      "name": "ZoneTemperatureHeatingOccupiedSetpointShape",
+      "types": ["air-temp-sp"],
+      "tags": ["zone", "heating", "occ"]
+    },
+    {
+      "name": "ZoneTemperatureCoolingUnoccupiedSetpointShape",
+      "types": ["air-temp-sp"],
+      "tags": ["zone", "cooling", "unocc"]
+    },
+    {
+      "name": "ZoneTemperatureHeatingUnoccupiedSetpointShape",
+      "types": ["air-temp-sp"],
+      "tags": ["zone", "heating", "unocc"]
+    },
+    {
+      "name": "ZoneTemperatureCoolingStandbySetpointShape",
+      "types": ["air-temp-sp"],
+      "tags": ["zone", "cooling", "standby"]
+    },
+    {
+      "name": "ZoneTemperatureHeatingStandbySetpointShape",
+      "types": ["air-temp-sp"],
+      "tags": ["zone", "heating", "standby"]
+    },
```

Comment:
Can you create a new function grouping for this based on the spreadsheet? For example, use these six shapes to create a new (composite) shape called: `ZoneSetpointsThermalComfort`. Then the vav would look something like:
```
      "name": "NREL-VAV-SD-Cooling-Only-Shape",
      "types": ["coolingOnly-vav"],
      "shape-mixins": ["ZoneSetpointsThermalComfort", ...other functional groups... ]
}
```

Then this becomes really modular / reusable for unitary equipment / etc.